### DISCUSSION TITLE
Azure DevOps: support Pull Request

### DISCUSSION THREAD

**Discussion by bobyang3** [gitextensions/gitextensions#10723] [Q&A]
### Feature description

I saw there is a Pull Request feature for GitHub. Is it possible to support Azure DevOps as well? thanks

### Environment

- GitExtensions version: 4.0.2.16100
- GIT version: 2.39.1
- OS version: Windows 10
- .NET version: 6.0.13

**Comment by bobyang3** [comment]
Hi @mstv,

Currently, we use internal Azure DevOps as Repo, not GitHub. It will be good if we can create/view pull requests by using gitextensions. thanks


![image](https://user-images.githubusercontent.com/2717137/218577461-fed06463-ec91-4db0-863d-c95dbad690f5.png)

**Comment by mstv** [comment]
Such integrations are implemented as plug-ins. I know of just https://github.com/gitextensions/GitExtensions.AzureDevOpsCommitMessage.
Perhaps you can use it as a template in order to create one for your need.

---

When pushing a new branch to GitHub, the git command prints an URL for creating an according pull request.

Alternatively, you could define a "Revision Link" (refer to https://git-extensions-documentation.readthedocs.io/en/release-4.0/settings.html#git-extensions-revision-links).
